components/insight/SRC/org/openmicroscopy/shoola/env/ui/OpenObjectLoader.java

```diff
@@ -93,14 +119,22 @@ public void load()
     	File f;
     	if (object instanceof ImageData) {
     		ImageData image = (ImageData) object;
-    		String name = image.getName();
-    		name += image.getName();
-    		name += "_"+image.getId();
-    		path += UIUtilities.replaceNonWordCharacters(name)+"."+OMETIFFFilter.OME_TIFF;
-    		f = new File(path);
-    		f.deleteOnExit();
-    		handle = ivView.exportImageAsOMETiff(ctx, image.getId(), f, null,
-    				this);
+    		if (originalImage) {
+    		    List<DataObject> objects = new ArrayList<DataObject>();
+    		    objects.add(image);
+    		    f = new File(folderPath);
```

Comment:
Will this `folderPath` get cleaned up on delete?

Reply:
I will need to check.
This is very old code

Reply:
Review the code. The file is stored in the specified tmp directory
Nothing was deleted on exit